### PR TITLE
Учебный проект: начинаем программировать

### PR DESCRIPTION
**Pull request by mfrswrkha**
Добрый день.  Внесены настройки мага в файл index.html

---
:mortar_board: [Учебный проект: начинаем программировать](https://up.htmlacademy.ru/javascript/20/user/607519/tasks/2)

:boom: https://htmlacademy-javascript.github.io/607519-code-and-magick-20/1/

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -180,7 +180,31 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
...
+if (isWindLeft) {
+    speed=5
+    } 
+return speed;
+}
+var wizardSpeed= 3;
+var wizardWidth=70;
+
+function getWizardHeight () {
+wizardHeight=1.337*wizardWidth;
```
Comment (alex-lancer) [inline index.html:196]: - You have defined **wizardHeight** as global variable. Use local variables.
- Please do not use _Magic Numbers_ Define constant for _1.337_ value

File: index.html [hunk 2]
```diff
@@ -180,7 +180,31 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
...
-
+<script>
+var fireballSize=22; 
+function  getFireballSpeed  (isWindLeft) {
+var speed=2    
+if (isWindLeft) {
+    speed=5
+    } 
+return speed;
+}
```
Comment (alex-lancer) [inline index.html:191]: You can use Conditional (ternary) operator hera and not override _speed_ value

Comment (mfrswrkha) [reply index.html:194]: I agree, but I used  the approach as in https://htmlacademy.ru/courses/215/run/14

File: index.html (hunk 1 above)
Comment (mfrswrkha) [reply index.html]: Ok

File: index.html [hunk 3]
```diff
@@ -182,24 +182,28 @@ <h4 class="setup-similar-title">Похожие персонажи</h4>
...
     speed=5
     } 
 return speed;
 }
-var wizardSpeed= 3;
-var wizardWidth=70;
 
 function getWizardHeight () {
-wizardHeight=1.337*wizardWidth;
+const ratioHeight =1.337;
```
Comment (alex-lancer) [inline index.html:197]: **const** is a ES6 feature. You cannot use it in this cource.

**Comment by alex-lancer** [comment]
Please update your project according to Lecture #2 recommendations

File: index.html (hunk 3 above)
Comment (mfrswrkha) [reply index.html]: update from const to var. Project was updated

**Pull request merged**